osu.Game/Overlays/Settings/Sections/UserInterface/MainMenuSettings.cs

```diff
@@ -43,5 +52,17 @@ private void load(OsuConfigManager config)
                 }
             };
         }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            backgroundSourceDropdown.Current.BindValueChanged(source =>
+            {
+                const string not_supporter_note = "Changes to this setting will only apply with an active osu!supporter tag.";
+
+                backgroundSourceDropdown.NoteText = user.Value?.IsSupporter != true ? not_supporter_note : string.Empty;
```

Comment:
Now that notes are a thing, I think it would be better UX to disable the dropdown if user doesn't have supporter, yeah?

Reply:
The user who isn't a supporter won't know the other options (and may put them a step back from getting supporter 😉).

Reply:
Ah, you're right, sounds like it should be a dropdown with unselectable items instead, but that's for later.